mapproxy/test/system/test_seed.py

```diff
@@ -236,46 +240,46 @@ def test_seed_mbtile(self):
         with tmp_image((256, 256), format='png') as img:
             img_data = img.read()
             expected_req = ({'path': r'/service?LAYERS=bar&SERVICE=WMS&FORMAT=image%2Fpng'
-                                  '&REQUEST=GetMap&VERSION=1.1.1&bbox=-180.0,-90.0,180.0,90.0'
-                                  '&width=256&height=128&srs=EPSG:4326'},
+                             '&REQUEST=GetMap&VERSION=1.1.1&bbox=-180.0,-90.0,180.0,90.0'
+                             '&width=256&height=128&srs=EPSG:4326'},
                             {'body': img_data, 'headers': {'content-type': 'image/png'}})
             with mock_httpd(('localhost', 42423), [expected_req]):
-                seed_conf  = load_seed_tasks_conf(self.seed_conf_file, self.mapproxy_conf)
+                seed_conf = load_seed_tasks_conf(self.seed_conf_file, self.mapproxy_conf)
                 tasks, cleanup_tasks = seed_conf.seeds(['mbtile_cache']), seed_conf.cleanups(['cleanup_mbtile_cache'])
                 seed(tasks, dry_run=False)
                 cleanup(cleanup_tasks, verbose=False, dry_run=False)
 
     def create_tile(self, coord=(0, 0, 0)):
         return Tile(coord,
-            ImageSource(tile_image_buf,
-                image_opts=ImageOptions(format='image/png')))
+                    ImageSource(tile_image_buf,
+                                image_opts=ImageOptions(format='image/png')))
 
     def test_reseed_mbtiles(self):
-        seed_conf  = load_seed_tasks_conf(self.seed_conf_file, self.mapproxy_conf)
-        tasks, cleanup_tasks = seed_conf.seeds(['mbtile_cache']), seed_conf.cleanups(['cleanup_mbtile_cache'])
+        seed_conf = load_seed_tasks_conf(self.seed_conf_file, self.mapproxy_conf)
+        tasks, _ = seed_conf.seeds(['mbtile_cache']), seed_conf.cleanups(['cleanup_mbtile_cache'])
```

Comment:
interesting variable rename?

Reply:
the variable is not used. if it is named `_` it is ignored by the linter.

Reply:
```suggestion
        tasks = seed_conf.seeds(['mbtile_cache'])
        seed_conf.cleanups(['cleanup_mbtile_cache'])
```